File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int cnt = 0;
        if (nums[nums.size() - 1] > nums[0]) cnt++;
        for (int i = 0; i < nums.size()-1; i++) {
            if (nums[i] > nums[i+1]) cnt++;
        }
        cout << cnt;
        if (cnt == 1 || cnt == 0) return true;
        else return false;
    }
};

File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int cnt = 0;
        if (nums[nums.length-1] > nums[0]) cnt++;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i-1] > nums[i]) {
                cnt++;
            }
        }
        if (cnt > 1) return false;
        else return true;
    }
}